docs/tutorial.rst

```diff
@@ -104,21 +104,27 @@ read :func:`dask_cassandra_loader.loader.Loader.load_cassandra_table`.
 
 .. code-block:: python
 
-  table = dask_cassandra_loader.load_cassandra_table('tab1',
+  try:
```

Comment:
Why are you wrapping it with a try block? The exception thrown by the load_cassandra_table method is already raised by the library itself and should be clear enough. For me it makes it harder to read the tutorial

Reply:
solved by c2f1205